R/pkg/R/mllib.R

```diff
@@ -533,6 +630,28 @@ setMethod("write.ml", signature(object = "KMeansModel", path = "character"),
             invisible(callJMethod(writer, "save", path))
           })
 
+# Saves the Multilayer Perceptron Classification Model to the input path.
+
+#' @param object A Multilayer Perceptron Classification Model fitted by \code{spark.mlp}
+#' @param path The directory where the model is saved
+#' @param overwrite Overwrites or not if the output path already exists. Default is FALSE
+#'                  which means throw exception if the output path exists.
+#'
+#' @rdname spark.mlp
+#' @aliases write.ml,MultilayerPerceptronClassificationModel,path-method
```

Comment:
I think this should be
`#' @aliases write.ml,MultilayerPerceptronClassificationModel,character-method`